README.md

```diff
@@ -1,38 +1,136 @@
 # BentoMap
+> Map Clustering for Swift
 
 [![Build Status](https://travis-ci.org/Raizlabs/BentoMap.svg?branch=develop)](https://travis-ci.org/Raizlabs/BentoMap)
 [![Version](https://img.shields.io/cocoapods/v/BentoMap.svg?style=flat)](http://cocoapods.org/pods/BentoMap)
 [![License](https://img.shields.io/cocoapods/l/BentoMap.svg?style=flat)](http://cocoapods.org/pods/BentoMap)
 [![Platform](https://img.shields.io/cocoapods/p/BentoMap.svg?style=flat)](http://cocoapods.org/pods/BentoMap)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 
-BentoMap is an implementation of [quadtrees](https://en.wikipedia.org/wiki/Quadtree) for map annotation clustering and storage written in Swift. The library intends to require minimal code in-app to get clustered annotations on screen, the example target included in BentoMap.xcodeproj contains the bare code required dynamically update the map with clusters whenever the map view's bounds changes.
+BentoMap is an implementation of [quadtrees][wiki] originally implemented for map annotation clustering and storage written in Swift. It expanded to allow other data that can be represented with 2D coordinates to conform to a protocol and be added into `BentoBox` containers.
```

Comment:
#nth the "written in Swift" is a misplaced modifier. It suggests that the map annotation clustering and storage is the thing that is written in Swift, which is true but is not the point you are trying to make here. Also, I take issue with the "originally implemented" because readers of the readme don't care about the history of the project - just what it can do for them right now.